wagtail/core/models.py

```diff
@@ -42,6 +42,9 @@
 
 
 class SiteManager(models.Manager):
+    def get_queryset(self):
+        return super(SiteManager, self).get_queryset().order_by(Lower("hostname"))
+
```

Comment:
Great central solution, love it!

Reply:
Thank you for the review. I love it when I get quick feedback!